testdata/baselines/reference/submodule/compiler/coAndContraVariantInferences2.types.diff

```diff
@@ -42,20 +42,17 @@
  
  function bar(node: Identifier | FunctionDeclaration) {
 ->bar : (node: Identifier | FunctionDeclaration) => void
-->node : Identifier | FunctionDeclaration
 +>bar : (node: FunctionDeclaration | Identifier) => void
```

Comment:
Curious difference; the function parameter thing should have changed but I guess Strada is printing unions in a node-preserving order still, which means this PR is not even the best example yet either due to the lack of node reuse in corsa.